dotcom-rendering/src/components/LatestLinks.importable.tsx

```diff
@@ -0,0 +1,199 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import {
+	type ArticleFormat,
+	timeAgo as timeAgoHasAWeirdInterface,
+} from '@guardian/libs';
+import { palette, space, textSans } from '@guardian/source-foundations';
+import { Hide } from '@guardian/source-react-components';
+import { decideContainerOverrides } from '../lib/decideContainerOverrides';
+import { decidePalette } from '../lib/decidePalette';
+import { revealStyles } from '../lib/revealStyles';
+import { useApi } from '../lib/useApi';
+import type { DCRContainerPalette } from '../types/front';
+import { WithLink } from './CardHeadline';
+
+type Props = {
+	id: string;
+	format: ArticleFormat;
+	direction: 'horizontal' | 'vertical';
+	isDynamo?: true;
+	containerPalette?: DCRContainerPalette;
+};
+
+type Blocks = {
```

Comment:
Possibly rename to make clear it's not content blocks? 
```suggestion
type LiveBlogBlocks = {
```

Reply:
Opted for inlining the type as the parameter to `useApi` so it’s clear that no actual type-checking happens